### DISCUSSION TITLE
Just curious about  Geo Distance Query

### DISCUSSION THREAD

**Discussion by joshfom** [Jeroen-G/Explorer#56] [General]
This is a great package, i just am just curious about Geo Distance Query  and sorting data by distance, 
[Elastic search geo query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-geo-distance-query.html), 
The project i am working on is Geospatial base, all results are based on user's location.

Please any heads up how to prepare the model index the document and perform search if it is possible

Thanks alot

**Comment by Jeroen-G** [comment]
I have never worked with spatial data and Elasticsearch so I won't be able to help you. Writing your own query is as easy as implementing the syntax interface.
